Cargo.lock

```diff
@@ -122,31 +122,52 @@ version = "0.11.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "69e32ef5c74bbeb1733c37f4ac7f866f8c8af208b7b4265e21af609dcac5bd5e"
 dependencies = [
- "alloy-eips",
+ "alloy-eips 0.11.1",
  "alloy-primitives",
  "alloy-rlp",
- "alloy-serde",
+ "alloy-serde 0.11.1",
  "alloy-trie",
- "arbitrary",
  "auto_impl",
  "c-kzg",
  "derive_more 1.0.0",
+ "serde",
+]
+
+[[package]]
+name = "alloy-consensus"
+version = "0.12.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "5758065889c3e6c7ba7bf0a2b02d0f635747074a5527e2a77ba2615ccf06748f"
+dependencies = [
+ "alloy-eips 0.12.3",
+ "alloy-primitives",
+ "alloy-rlp",
+ "alloy-serde 0.12.3",
+ "alloy-trie",
+ "arbitrary",
+ "auto_impl",
+ "c-kzg",
+ "derive_more 2.0.1",
+ "either",
  "k256",
+ "once_cell",
```

Comment:
Looks like ci is failing because `once_cell` was [introduced in alloy-consensus](https://github.com/alloy-rs/alloy/blob/main/crates/consensus/Cargo.toml#L56C1-L56C10) with "critical-section" feature enabled.

Reply:
should be fixed after https://github.com/alloy-rs/alloy/pull/2165